batch/batch/front_end/front_end.py

```diff
@@ -392,7 +392,7 @@ async def get_completed_batches_ordered_by_completed_time(request, userdata):
     wheres = [
         'billing_project_users.`user` = %s',
         'billing_project_users.billing_project = batches.billing_project',
-        'time_completed IS NOT NULL',
+        'batches.time_completed IS NOT NULL',
```

Comment:
If this is necessary, then we probably also need `NOT batches.deleted` on the next line and for `time_completed` on the `wheres.append` and `ORDER BY time_completed DESC` lines later in the function.